tests/ci/run_benchmark_build_tests.sh

```diff
@@ -13,47 +13,54 @@ openssl_1_0_branch='OpenSSL_1_0_2-stable'
 # failing our CI.
 openssl_3_0_branch='openssl-3.0.5'
 
+function build_aws_lc_fips {
+  echo "building aws-lc in FIPS mode"
+  run_build -DCMAKE_INSTALL_PREFIX="${install_dir}/aws-lc-fips" -DFIPS=1
+  pushd "$BUILD_ROOT"
+  ninja install
+  popd
+}
+
 function build_openssl_1_0 {
     echo "building OpenSSL 1.0"
-    git clone --branch "${openssl_1_0_branch}" "${openssl_url}" "../openssl-1.0"
+    git clone --depth 1 --branch "${openssl_1_0_branch}" "${openssl_url}" "../openssl-1.0"
     pushd "../openssl-1.0"
     mkdir -p "${install_dir}/openssl-1.0"
     ./config --prefix="${install_dir}/openssl-1.0" --openssldir="${install_dir}/openssl-1.0"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-1.0"
 }
 
 function build_openssl_1_1 {
     echo "building OpenSSL 1.1"
-    git clone --branch "${openssl_1_1_branch}" "${openssl_url}" "../openssl-1.1"
+    git clone --depth 1 --branch "${openssl_1_1_branch}" "${openssl_url}" "../openssl-1.1"
     pushd "../openssl-1.1"
     mkdir -p "${install_dir}/openssl-1.1"
     ./config --prefix="${install_dir}/openssl-1.1" --openssldir="${install_dir}/openssl-1.1"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-1.1"
 }
 
 function build_openssl_3_0 {
     echo "building OpenSSL 3.0"
-    git clone --branch "${openssl_3_0_branch}" "${openssl_url}" "../openssl-3.0"
+    git clone --depth 1 --branch "${openssl_3_0_branch}" "${openssl_url}" "../openssl-3.0"
     pushd "../openssl-3.0"
     mkdir -p "${install_dir}/openssl-3.0"
     ./Configure --prefix="${install_dir}/openssl-3.0" --openssldir="${install_dir}/openssl-3.0"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-3.0"
 }
 
 # We build each tool individually so we can have more insight into what is failing
-mkdir -p "${install_dir}"
-echo "Testing awslc_bm"
-mkdir -p "${install_dir}/aws_lc"
-run_build -DAWSLC_INSTALL_DIR="${install_dir}/aws_lc"
```

Comment:
We probably want to keep the test for aws-lc building in non-fips mode

Reply:
I don't think this test was doing anything, it was just building the benchmark with the "local" aws-lc: there was nothing in `${install_dir}/aws_lc`. We already build the normal benchmark on every build. What I think the spirit of this test was supposed to be is:
1. build aws-lc's libcrypto somewhere somehow
2. copy that libcrypto
3. build the latest and greatest benchmark with that aws-lc libcrypto